website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,199 @@ description: |-
+network segment boundaries into "network segments": isolated LAN gossip pools
+with full connectivity between all agent members.
+
+![Consul datacenter agent connectivity with network segments](/img/network-segments/consul-network-segments-multiple.png)
+
+To get started with network segments you can review the tutorial on HashiCorp
+Learn for [Network Segments](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters).
+
+-> **Info:** Network segments enable operating a datacenter without full mesh
+(LAN) connectivity between agents. To enable operating multiple federated Consul
+datacenters without full mesh (WAN) connectivity between server agents, use
+[network areas (Enterprise)](https://www.consul.io/api-docs/operator/area) instead.
+
+## Server Configuration
+
+Server agents are members of all segments and therefore provide a connection
+point across segments. Segments include (1) the `<default>` segment and (2) the
+additional segments specified in their [`segments`](https://www.consul.io/docs/agent/options.html#segments)
+agent configuration option. Each additional segment specifies a non-empty name
+and unique port, and optionally a custom bind and advertise address, for that
+segment's listener on the server.
+
+~> **Note:** Prior to Consul 1.7.3, a Consul server agent configured with too
+many network segments may not be able to start due to [limitations](https://learn.hashicorp.com/tutorials/consul/network-partition-datacenters#network-segments-limitations)
+in Serf.
+
+### Example Server Configuration
+
+If the snippet below is added to a server agent configuration file, it will
+create two user-defined segments: `alpha` and `beta`.
+
+<Tabs>
+<Tab heading="HCL">
+
+```hcl
+segments= [
+  {
+    name= "alpha"
+    bind= "{{GetPrivateIP}}"
+    advertise= "{{GetPrivateIP}}"
+    port= 8303
+  },
+  {
+    name= "beta"
+    bind= "{{GetPrivateIP}}"
+    advertise= "{{GetPrivateIP}}"
+    port= 8304
+  }
+]
+```
+
+</Tab>
+<Tab heading="JSON">
+
+```json
+{
+  "segments": [
+    {
+      "name": "alpha",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8303
+    },
+    {
+      "name": "beta",
+      "bind": "{{GetPrivateIP}}",
+      "advertise": "{{GetPrivateIP}}",
+      "port": 8304
+    }
+  ]
+}
+```
+
+</Tab>
+</Tabs>
+
+List of relevant server configuration parameters, all within the server
+[agent configuration](https://www.consul.io/docs/agent/options):
+
+- [`ports.serf_lan`](https://www.consul.io/docs/agent/options#serf_lan_port):
+  listener port for `<default>` segment on this server
+- [`segments`](https://www.consul.io/docs/agent/options#segments): for
+  configuring user-defined segments on this server
+
+## Client Configuration
+
+Client agents are members of only one segment, which will be the `<default>`
+segment unless otherwise specified in their [`segment`](https://www.consul.io/docs/agent/options.html#_segment)
+agent configuration option.
```

Comment:
Changed to

```
Each client agent is a member of only one segment at a time. This will be the
`<default>` segment unless otherwise specified in their [`segment`](/docs/agent/options.html#_segment)
agent configuration option.
```